helm/url-shortener/templates/frontend-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  namespace: url-shortener
```

Comment:
this could be defined on custom-values file, I would define the namespace on the chart values